src/app/component/layout/components/header/header.component.spec.ts

```diff
@@ -48,6 +49,7 @@ describe('HeaderComponent', () => {
   let userServiceMock: UserService;
   userServiceMock = jasmine.createSpyObj('UserService', ['onLogout']);
   userServiceMock.onLogout = () => true;
+  userServiceMock.updateUserLanguage = () => Observable.of();
```

Comment:
use of() instead of Observable.of()
also add value, like so of(true)

Reply:
fixed